.github/workflows/test.yaml

```
@@ -0,0 +1,20 @@
+name: Test
+on:
+  push
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Test Drupal module
+        uses: actions/checkout@v2
+        with:
+          repository: discoverygarden/embargoes
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          path: .github/actions/CodeSniffer
+
```

```suggestion
      - name: Run CodeSniffer
        uses: ./.github/actions/CodeSniffer
        with:
          path: clean
     - name: Run CodeSniffer
        uses: ./.github/actions/CodeSniffer
        with:
          path: fail-lint
```
Could add multiple runs with directories of code to test different cases.